contentstore/client.go

```diff
@@ -45,3 +51,12 @@ func (p *ContentStoreClient) DoRequest(httpMethod string, path string, data []by
 
 	return p.client.Do(req)
 }
+
+// FIXME: remove once we have a draft content store wherever publishing-api is present.
+func emptyOKResponse() *http.Response {
+	return &http.Response{
+		Status:     "200 OK",
+		StatusCode: 200,
+		Body:       ioutil.NopCloser(bytes.NewBufferString("Draft content-store not configured. This is a dummy response")),
```

Comment:
Might be simpler to use a [`strings.Reader`](http://golang.org/pkg/strings/#Reader) here.